docs/source/model_sharing.rst

```diff
@@ -111,13 +111,33 @@ Make sure there are no garbage files in the directory you'll upload. It should o
 - a `pytorch_model.bin` file, which is the PyTorch checkpoint (unless you can't have it for some reason) ;
 - a `tf_model.h5` file, which is the TensorFlow checkpoint (unless you can't have it for some reason) ;
 - a `special_tokens_map.json`, which is part of your :doc:`tokenizer <main_classes/tokenizer>` save;
-- a `tokenizer_config.json`, which is part of your :doc:`tokenizer <main_classes/tokenizer>` save;
+- a `tokenizer_config.json`, which is part of your :doc:`tokenizerawesome-name-you-picked <main_classes/tokenizer>` save;

```

Comment:
unintended change?

Reply:
I have no idea how this got there, good catch!